src/flags.rs

```diff
@@ -98,10 +98,11 @@ pub trait Configurable<T>
 where
     T: std::default::Default,
 {
-    /// Returns a value from either [ArgMatches], a [Config] or a [Default] value. The first value
-    /// that is not [None] is used. The order of precedence for the value used is:
+    /// Returns a value from either [ArgMatches], a [Config], a [Default] or the environment value.
+    /// The first value that is not [None] is used. The order of precedence for the value used is:
     /// - [from_arg_matches](Configurable::from_arg_matches)
     /// - [from_config](Configurable::from_config)
+    /// - [from_environment](Configurable::from_environment)
```

Comment:
I am thinking the order could be arg -> env -> config. What do you think?

Reply:
I wasn't sure about the correct order to use. If you'd like to change the order around I can do that.

Reply:
I think we  will go with arg -> env -> config .